device/arista/x86_64-arista_7050_qx32/platform.json

```diff
@@ -1,89 +1,63 @@
 {
     "chassis": {
-        "name": "DCS-7050QX-32",
         "components": [],
-        "fans": [
-            {
-                "name": "fan1"
-            },
-            {
-                "name": "fan2"
-            },
-            {
-                "name": "fan3"
-            },
-            {
-                "name": "fan4"
-            }
-        ],
         "fan_drawers": [
             {
-                "name": "fan1",
                 "fans": [
                     {
                         "name": "fan1"
                     }
-                ]
+                ],
+                "name": "fan1"
             },
             {
-                "name": "fan2",
                 "fans": [
                     {
                         "name": "fan2"
                     }
-                ]
+                ],
+                "name": "fan2"
             },
             {
-                "name": "fan3",
                 "fans": [
                     {
                         "name": "fan3"
                     }
-                ]
+                ],
+                "name": "fan3"
             },
             {
-                "name": "fan4",
                 "fans": [
                     {
                         "name": "fan4"
                     }
-                ]
-            }
-        ],
-        "psus": [
-            {
-                "name": "psu1",
-                "fans": []
-            },
-            {
-                "name": "psu2",
-                "fans": []
+                ],
+                "name": "fan4"
             }
         ],
-        "thermals": [
-            {
-                "name": "Cpu temp sensor"
-            },
-            {
-                "name": "Board sensor"
-            },
+        "fans": [
             {
-                "name": "Front-panel temp sensor"
+                "name": "fan1"
             },
             {
-                "name": "Rear temp sensor"
+                "name": "fan2"
             },
             {
-                "name": "Power supply 1 inlet temp sensor"
+                "name": "fan3"
             },
             {
-                "name": "Power supply 1 internal sensor"
-            },
+                "name": "fan4"
+            }
+        ],
+        "name": "DCS-7050QX-32",
```

Comment:
I'm not a fan of moving the "name" line down, but you must have generated this file with a script. Are the changes in other areas in line with what sonic-mgmt testing expects? Granted we haven't tested with this platform in a while so for sure some of this data here is outdated.

Reply:
I can minimize the diff here if preferred - it was indeed generated this file based on port_config.ini (https://gist.github.com/bluecmd/15c05daf2e5f0662928220886e827b03) together with some manual changes.

>  Are the changes in other areas in line with what sonic-mgmt testing expects?

No idea! How do I check? :-)

Reply:
Diff has been minimized, do you want more changes here?